shared/js/media/media_utils.js

```diff
@@ -19,6 +21,51 @@ var MediaUtils = {
     });
   },
 
+  formatExifDate: function(dateString) {
+    var [y, m, d] = dateString.split(exifDateSplitRe, 3);
+
+    return new Date(y, m - 1, d).toLocaleString(navigator.languages, {
+      'month': 'numeric',
+      'year': 'numeric',
+      'day': 'numeric'
+    });
+  },
+
+  formatExifFlash: function(flashValue) {
+    var FLASH_FIRED_MASK = 0x1;
+//    var FLASH_RETURN_MASK = 0x6;
+    var FLASH_MODE_MASK = 0x18;
+//    var FLASH_FUNCTION_MASK = 0x20;
+    var FLASH_REDEYE_MASK = 0x40;
+
+    if(flashValue === 0) {
+      return ['flash-none'];
+    }
+
+    var values = [];
+
+    switch (flashValue & FLASH_MODE_MASK) {
+    case 0x08:
+      values.push('flash-on');
+      break;
+    case 0x10:
+      values.push('flash-off');
+      break;
+    case 0x18:
+      values.push('flash-auto');
+      break;
+    }
+    if (flashValue & FLASH_FIRED_MASK) {
+      values.push('flash-fired');
+    } else {
+      values.push('flash-not-fired');
+    }
+    if (flashValue & FLASH_REDEYE_MASK) {
+      values.push('flash-red-eye');
+    }
+    return values;
+  },
```

Comment:
Could you do the localization in this file using the new formatValue rather than doing it in info.js using the old get() function?